fenics_helpers/timestepping.py

```diff
@@ -6,42 +6,39 @@ class Progress:
     def __init__(self, t_start, t_end, show_bar):
         self._show_bar = show_bar
 
-        self.TGREEN = "\033[32m"
-        self.TRED = "\033[31m"
-        self.TEND = "\033[m"
-
         if self._show_bar:
             from tqdm import tqdm
 
-            self._pbar = tqdm(
-                total=t_end - t_start, ascii=True, bar_format="{l_bar}{bar}{rate_fmt}"
-            )
+            fmt = "{l_bar}{bar}{rate_fmt}"
+            self._pbar = tqdm(total=t_end - t_start, ascii=True, bar_format=fmt)
+
+    def _green(self, msg):
+        info("\033[32m" + msg + "\033[m")
```

Comment:
Not sure I like these methods. For one, they aren't really methods (as in `self` is not used), so they probably should be free functions. For two, they do two things at once, make stuff green and print.

I'd say we either just use string concatenation (no need to have this in this class though)
~~~python
RED = "\033[31m"
GREEN = "\033[32m"
RESET = "\033[m"
print(RED + msg + RESET)
~~~
or write a small function so that you can do
~~~python
print(colored(msg, "red"))
~~~